cpp/include/generic_proxy_client.h

```diff
@@ -112,6 +112,10 @@ class GenericProxy_client : public Proxy_client {
 
   virtual bool has_local_fragment_cache();
 
+  virtual boost::optional<string>
+  get_fragment_encryption_key(const string &alba_id,
```

Comment:
I named it like this because the property in the preset is named "fragment_encryption" ... but I'm not attached to the name